### PR TITLE
Add TTM sentence parse support

### PR DESCRIPTION
**Pull request by Tapped**
https://github.com/AeroRust/nmea/issues/54 Add TTM sentence parse support

TTM is a stand-alone feature, but part of a new feature called "radar", when other radar messages as TLL or TLB is introduced.

### CODE REVIEW

File: src/sentences/ttm.rs [hunk 1]
```diff
@@ -11,6 +12,7 @@ use serde::{Deserialize, Serialize};
 
 use super::utils::{parse_float_num, parse_hms, parse_number_in_range};
 use crate::{Error, NmeaSentence, SentenceType};
+use heapless;
```
Comment (elpiel) [inline src/sentences/ttm.rs:15]: that's actually not needed  as an import. I would suggest importing `heapless::String` directly.

**Review by elpiel — COMMENTED** [review]
Thank you for your PR! Looks good to me!

Could you do 2 more things to complete it:
1. Add the example sentence in the `TtmData` docs as `Example` (you can check some of the other sentences for this
2. Add the example in the https://github.com/AeroRust/nmea/blob/main/tests/all_supported_messages.rs test

**Comment by Tapped** [comment]
@elpiel Thank you, I fixed your suggestions!

**Comment by elpiel** [comment]
> @elpiel Thank you, I fixed your suggestions!

I'm not sure if you are aware but the CI fails.

**Comment by Tapped** [comment]
@elpiel 
Yeah I noticed, but the errors seems to be unrelated to this PR.

I suspect its caused by an update of clippy. I'll make a separate PR to fix those errors, that I assume fails on latest main as well. 

See for example:
https://github.com/AeroRust/nmea/actions/runs/9164588924/job/25378071147?pr=128#step:5:65

**Comment by Tapped** [comment]
I'm not sure what to do with the codecov failing?

**Comment by elpiel** [comment]
Nothing, I will find some time to investigate it.

Thank you once again for your contributions and patience!


If you are looking for more fun on the nmea crate, there are a few key features that will make the crate very useful for embedded platforms.

**Pull request merged**